active: false
main_timer_rate: 100
max_distance: 100

speed: 4.0
acceleration: 2.0
jerk: 3.0

height: 1.5
randomize_distance: true

trajectory_dt: 0.2
